client.go

```diff
@@ -199,6 +202,22 @@ func (c *Client) CreateSession(cfg *uasc.SessionConfig) (*Session, error) {
 	return s, err
 }
 
+const defaultAnonymousPolicyID = "Anonymous"
+
+func anonPolicyID(endpoints []*ua.EndpointDescription) string {
+	for _, e := range endpoints {
+		if e.SecurityMode == ua.MessageSecurityModeNone && e.SecurityPolicyURI == ua.SecurityPolicyURINone {
```

Comment:
Two nitpicks:

1. Either `anon` or `Anonymous` but not both
1. How about flipping the `if` to avoid one level of nesting?

```go
for _, e := range endpoints {
    if e.SecurityMode != ua.MessageSecurityModeNone || 
        e.SecurityPolicyURI != ua.SecurityPolicyURINone {
        continue
    }
    for _, t := range e.UserIdentityToken { ... }
}
```